docs/sources/flow/reference/components/otelcol.receiver.aws_firehose.md

```diff
@@ -0,0 +1,165 @@
+---
+aliases:
+- /docs/grafana-cloud/agent/flow/reference/components/otelcol.receiver.aws_firehose/
+- /docs/grafana-cloud/monitor-infrastructure/agent/flow/reference/components/otelcol.receiver.aws_firehose/
+- /docs/grafana-cloud/monitor-infrastructure/integrations/agent/flow/reference/components/otelcol.receiver.aws_firehose/
+- /docs/grafana-cloud/send-data/agent/flow/reference/components/otelcol.receiver.aws_firehose/
```

Comment:
New topic? We don't really need to add all these aliases. We only need them if the topic moves or is renamed after it has been published at least once. It's not hurting anything to add them, but they aren't necessary.